test/units/testsuite-58.sh

```diff
@@ -133,6 +133,39 @@ SizeMaxBytes=64M
 PaddingMinBytes=92M
 EOF
 
+    systemd-repart --definitions="$defs" \
+                   --dry-run=no \
+                   --seed="$seed" \
+                   --include-partitions=home \
+                   "$imgs/zzz"
+
+    output=$(sfdisk -d "$imgs/zzz" | grep -v -e 'sector-size' -e '^$')
+
+    assert_eq "$output" "label: gpt
+label-id: 1D2CE291-7CCE-4F7D-BC83-FDB49AD74EBD
+device: $imgs/zzz
+unit: sectors
+first-lba: 2048
+last-lba: 2097118
+$imgs/zzz1 : start=        2048, size=      591856, type=933AC7E1-2EB4-4F13-B844-0E14E2AEF915, uuid=4980595D-D74A-483A-AA9E-9903879A0EE5, name=\"home-first\", attrs=\"GUID:59\""
+
+    systemd-repart --definitions="$defs" \
+                   --dry-run=no \
+                   --seed="$seed" \
+                   --exclude-partitions=root \
```

Comment:
for added coverage would be good to pass an actual list to one of these, to ensure that's handled correctly